pkg/statistics/histogram.go

```diff
@@ -87,9 +87,13 @@ const EmptyHistogramSize = int64(unsafe.Sizeof(Histogram{}))
 
 // Bucket store the bucket count and repeat.
 type Bucket struct {
-	Count  int64
+	// Count is the number of items in the bucket.
+	Count int64
+	// Repeat is the number of repeats of the upper bound of the bucket.
```

Comment:
how about using **_right_** bound of this bucket range [x,**_y_**], more visually clear